install_config/configuring_nsxtsdn.adoc

```diff
@@ -0,0 +1,209 @@
+openshift_master_cluster_public_hostname=master01.example.com
+openshift_hosted_manage_registry=true
+openshift_hosted_manage_router=true
+openshift_enable_service_catalog=true
+openshift_cluster_monitoring_operator_install=true
+openshift_web_console_install=true
+openshift_console_install=true
+
+# NSX-T specific configuration
+#nsx_use_loadbalancer=false
+nsx_openshift_cluster_name='cluster01'
+nsx_api_managers='nsxmgr.example.com'
+nsx_api_user='nsx_admin'
+nsx_api_password='nsx_api_password_example'
+nsx_tier0_router='LR-Tier-0'
+nsx_overlay_transport_zone='TZ-Overlay'
+nsx_container_ip_block='pod-networking'
+nsx_no_snat_ip_block='pod-nonat'
+nsx_external_ip_pool='pod-external'
+nsx_top_fw_section='containers-top'
+nsx_bottom_fw_section='containers-bottom'
+nsx_ovs_uplink_port='ens224'
+nsx_cni_url='http://websrv.example.com/nsx-cni-buildversion.x86_64.rpm'
+nsx_ovs_url='http://websrv.example.com/openvswitch-buildversion.rhel75-1.x86_64.rpm'
+nsx_kmod_ovs_url='http://websrv.example.com/kmod-openvswitch-buildversion.rhel75-1.el7.x86_64.rpm'
+nsx_insecure_ssl=true
+
+# vSphere Cloud Provider
+#openshift_cloudprovider_kind=vsphere
+#openshift_cloudprovider_vsphere_username='administrator@example.com'
+#openshift_cloudprovider_vsphere_password='viadmin_password'
+#openshift_cloudprovider_vsphere_host='vcsa.example.com'
+#openshift_cloudprovider_vsphere_datacenter='Example-Datacenter'
+#openshift_cloudprovider_vsphere_cluster='example-Cluster'
+#openshift_cloudprovider_vsphere_resource_pool='ocp'
+#openshift_cloudprovider_vsphere_datastore='example-Datastore-name'
+#openshift_cloudprovider_vsphere_folder='ocp'
+
+[masters]
+master01.example.com
+master02.example.com
+master03.example.com
+
+[etcd]
+master01.example.com
+master02.example.com
+master03.example.com
+
+[nodes]
+master01.example.com ansible_ssh_host=192.168.220.2 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.2
+master02.example.com ansible_ssh_host=192.168.220.3 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.3
+master03.example.com ansible_ssh_host=192.168.220.4 OpenShift_node_group_name='node-config-master' OpenShift_ip=192.168.220.4
+node01.example.com ansible_ssh_host=192.168.220.5 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.5
+#node02.example.com ansible_ssh_host=192.168.220.6 OpenShift_node_group_name='node-config-infra' OpenShift_ip=192.168.220.6
+node03.example.com ansible_ssh_host=192.168.220.7 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.7
+node04.example.com ansible_ssh_host=192.168.220.8 OpenShift_node_group_name='node-config-compute' OpenShift_ip=192.168.220.8
+----
+
+Check prerequisites for {product-title} with NSX-T CNI:
+----
+$ ansible-playbook -i hosts openshift-ansible/playbooks/prerequisites.yml
+----
+
+Once the above playbook has executed successfully, run the following on all nodes:
+
+Assuming NCP Container image is downloaded locally on all nodes.
```

Comment:
@RobbieJVMW Does the OpenShift installer download these images or is this a manual step? 

| docker load -i nsx-ncp-rhel-xxx.tar
Is `nsx-ncp-rhel-xxx` a literal string to enter or does the user need to know the file name? If the later, how does the user know the file name?

Reply:
This is a manual set with 'nsx-ncp-rhel-xxx' where xxx refers to the NCP build version.